client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -487,10 +496,11 @@ protected String getPath(Method method, Object[] arguments, JaxRsMeta meta) {
 
         StringBuilder  query       = null;
         Class<?>[]     types       = method.getParameterTypes();
-        Annotation[][] annotations = method.getParameterAnnotations();
-        for (int i = 0; i < types.length; i++) {
-            Object value = arguments[i];
-            for (Annotation annotation : annotations[i]) {
+        List<Object> args = new ArrayList<>(asList(arguments));
```

Comment:
args = new ArrayList<>(asList(arguments));
maybe just 
args = asList(arguments);

Reply:
Will not work if we do :
"args.add(v.getValue1().apply(value));"

asList returns immutable list

Reply:
Yes, would have been nice, but then I can't add new vales to the list, which happens when a @BeanParam is discovered, as that param will map one argument to multiple.